docs/resources/logdna_alert.md

```diff
@@ -0,0 +1,117 @@
+
+- `emails`: _(Required)_ An array of email addresses (each email is of type _string_) to notify in the Alert
+- `immediate`: _(Optional)_ Whether the Alert will trigger immediately after the trigger limit is reached. _Note: Immediate can only be set to `true` for presence alerts_. Valid options are `true` and `false`, type _string_. (**Default: "false"**)
+- `operator`: _(Optional)_ Whether the Alert will trigger on the presence or absence of logs. Valid options are `presence` and `absence`, type _string_ (**Default: "presence"**)
+- `terminal`: _(Optional)_ Whether the Alert will trigger after the `triggerinterval` if the Alert condition is met (e.g., send an Alert after 30s). Valid options are `true` and `false`, type _string_ (**Default: "true"**)
+- `timezone`: _(Optional)_ Which time zone the log timestamps will be formatted in. Timezones are represented as [database time zones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones), type _string_
+- `triggerinterval`: _(Optional)_ Interval which the Alert will be looking for presence or absence of log lines. For presence Alerts, valid options are: `30`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. For absence Alerts, valid options are: `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. Type _string_ (**Defaults: "30" for presence; "15m" for absence**)
+- `triggerlimit`: _(Required)_ Number of lines before the Alert is triggered. (eg. Setting a value of `10` for an `absence` Alert would alert you if `10` lines were not seen in the `triggerinterval`), type _integer_
+
+### pagerduty_channel
+
+`pagerduty_channel` supports the following arguments:
+
+- `immediate`: _(Optional)_ Whether the Alert will trigger immediately after the trigger limit is reached. _Note: Immediate can only be set to `true` for presence alerts_. Valid options are `true` and `false`, type _string_ (**Default: "false"**)
+- `key`: _(Required)_ PagerDuty service key, type _string_
+- `operator`: _(Optional)_ Whether the Alert will trigger on the presence or absence of logs. Valid options are `presence` and `absence`, type _string_ (**Default: "presence"**)
+- `terminal`: _(Optional)_ Whether the Alert will trigger after the `triggerinterval` if the Alert condition is met (e.g., send an Alert after 30s). Valid options are `true` and `false`, type _string_ (**Default: "true"**)
+- `triggerinterval`: _(Optional)_ Interval which the Alert will be looking for presence or absence of log lines. For presence Alerts, valid options are: `30`, `1m`, `5m`, `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. For absence Alerts, valid options are: `15m`, `30m`, `1h`, `6h`, `12h`, and `24h`. Type _string_ (**Defaults: "30" for presence; "15m" for absence**)
+- `triggerlimit`: _(Required)_ Number of lines before the Alert is triggered. (eg. Setting a value of `10` for an `absence` Alert would alert you if `10` lines were not seen in the `triggerinterval`), type _integer_
+
+### webhook_channel
```

Comment:
@dm36, I think, instead of copying channels' docs, it's better to have a separate docs for just channels, no? What do you think, @tanberry?

Reply:
In general, yes, you want to have one, single source-of-truth, (never duplicated content) so that you only have to update docs in one place.

But let me make sure I understand what is here... is line 80 - 115 copied form another .md file in this repo?

Reply:
so it's the same for logdna_view.md and logdna_alert.md. I think in this context the duplication is ok- what would be the alternative?

Reply:
@smusali, @tanberry, not sure how good it may sound but having `alert_channels.md` and referencing it in both `logdna_alert.md` and `logdna_view.md` is what I have in my mind but it applies to only 2 resources

Reply:
how would it fall into this layout here: https://www.terraform.io/docs/registry/providers/docs.html?

A link alert_channel.md in docs/guides

Reply:
@dm36 how volatile is that content? Will it change with every release? If volatile, we should do as @smusali suggests, and create an alert_channels.md file (which would live in docs/guides), and then link to it from both logdna_alert.md and logdna_view.md.

If the content is uber-stable, we could live risky and leave it duplicated...

Reply:
it's unchanging- but i personally feel better about keeping it as it is rn over the docs/guides idea